contracts/src/Gateway.sol

```diff
@@ -650,7 +631,6 @@ contract Gateway is IGateway, IInitializable, IUpgradable {
         // Initialize agent for BridgeHub
         address bridgeHubAgent = address(new Agent(BRIDGE_HUB_AGENT_ID));
         core.agents[BRIDGE_HUB_AGENT_ID] = bridgeHubAgent;
-        core.agentAddresses[bridgeHubAgent] = BRIDGE_HUB_AGENT_ID;
```

Comment:
I think we should rather leave these in, in case we need this data later. I also understand we can't remove the [storage mapping](https://github.com/Snowfork/snowbridge/blob/main/contracts/src/storage/CoreStorage.sol#L16) without a migration, so it might not be worthwhile to remove this in the first place.

Reply:
Also dont we still need agents for Transact support?

Reply:
Probably yeah. I'll revert these.

Reply:
Reverted in https://github.com/Snowfork/snowbridge/pull/1317/commits/d15b6dfe69ad7ec4764e64946ddc5bb28442edd5.